data_summaries/Medical Meadow.json

```diff
@@ -0,0 +1,455 @@
+            "Medical Question Answering",
+            "Question Answering"
+        ],
+        "Text Sources": [],
+        "Model Generated": [
+            "OpenAI GPT-3.5-turbo"
+        ],
+        "Format": [
+            "Zero-shot"
+        ],
+        "Human Annotation": "No",
+        "Derived from Datasets": [
+            "PubMed Causal Language Use"
+        ],
+        "Creators": [
+            "University Hospital Aachen",
+            "Technische Universität München",
+            "Berliner Hochschule für Technik"
+        ],
+        "Licenses": [
+            {
+                "License": "GNU General Public License v3.0",
+                "License URL": "https://github.com/junwang4/causal-language-use-in-science/blob/master/LICENSE"
+            }
+        ],
+        "License Notes": "The license was provided in the github repo of the original dataset",
+        "License Verified By": "",
+        "Dataset Filter IDs": [
+            "medical-meadow-health-advice"
+        ]
+    },
+    "medical-meadow-pubmed-causal": {
+        "Unique Dataset Identifier": "medical-meadow-pubmed-causal",
+        "Dataset Name": "pubmed-causal",
+        "Paper Title": "Detecting Causal Language Use in Science Findings",
+        "Dataset URL": "https://aclanthology.org/D19-1473/",
+        "GitHub URL": "https://github.com/junwang4/causal-language-use-in-science",
+        "Hugging Face URL": "https://huggingface.co/datasets/medalpaca/medical_meadow_pubmed_causal",
+        "Papers with Code URL": "",
+        "ArXiv URL": "",
+        "Semantic Scholar Corpus ID": 202787823,
+        "Collection": "Medical Meadow",
+        "Collection URL": "https://github.com/kbressem/medAlpaca",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "BioMedical Question Answering",
+            "Instructional Question Answering",
+            "Medical Question Answering",
+            "Question Answering"
+        ],
+        "Text Sources": [],
+        "Model Generated": [
+            "OpenAI GPT-3.5-turbo"
+        ],
+        "Format": [
+            "Zero-shot"
+        ],
+        "Human Annotation": "No",
+        "Derived from Datasets": [
+            "PubMed Causal Language Use"
+        ],
+        "Creators": [
+            "University Hospital Aachen",
+            "Technische Universität München",
+            "Berliner Hochschule für Technik"
+        ],
+        "Licenses": [
+            {
+                "License": "GNU General Public License v3.0",
+                "License URL": "https://github.com/junwang4/causal-language-use-in-science/blob/master/LICENSE"
+            }
+        ],
+        "License Notes": "The license was provided in the github repo of the original dataset",
+        "License Verified By": "",
+        "Dataset Filter IDs": [
+            "medical-meadow-pubmed-causal"
+        ]
+    },
+    "medical-meadow-mmlu": {
```

Comment:
Is this equivalent to the MMLU test set? I wonder if we should leave this out, just because its such a common test set. We omitted it from Flan SNI and other sources for that reason.

Reply:
I compared a few examples from this dataset to MMLU test, and yes, unfortunately it looks like the examples come from it. I think we should leave it out, too. So, do I exclude it?